### ISSUE TITLE
Support for Oracle Version Provider

### ISSUE THREAD

**Issue by LordCheese** [canton7/Simple.Migrations#6]
Are there any plans to write an Oracle Version Provider as well?

**Comment by canton7** [comment]
I don't use Oracle myself, so I'm not qualified to write one.

You can write your own easily: subclass [`VersionProviderBase`](https://github.com/canton7/Simple.Migrations/blob/develop/src/Simple.Migrations/VersionProviderBase.cs) and implement the three abstract methods to return suitable SQL. See the other providers for examples.

If you contribute it back, I'll happily include it :smile:

**Comment by LordCheese** [comment]
Haha sounds good, I'd be happy to poke away at that. I just didn't want to duplicate work already being done!

**Comment by canton7** [comment]
Sure, thanks for checking!

**Comment by canton7** [comment]
Hi, have you had a chance to look at writing an Oracle Version Provider?

**Comment by canton7** [comment]
Aha fantastic!

**Comment by canton7** [comment]
Hi, any news?

**Comment by brookpatten** [comment]
I have a working oracle provider I've been using if anyone is interested.  One possible point of contention is I use the dotconnect oracle driver (because it has actual async support, unlike odp.net).  If someone is interested I can PR it.

**Comment by canton7** [comment]
Yes please! We can call it something which indicates it uses the dotconnect driver, in case someone wants to write one which uses another driver.

**Comment by waldimen** [comment]
I implemented Oracle Provider using MangedOracleDriver, if you are intrested I can share it.

**Comment by canton7** [comment]
Please do!
